src/sentry/integrations/github/payload.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+
+import hashlib
+import hmac
+import logging
+import six
+
+from django.http import HttpResponse
+from django.utils.crypto import constant_time_compare
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from simplejson import JSONDecodeError
+from sentry import options
+
+from sentry.utils import json
+from sentry.api.base import Endpoint
+
+logger = logging.getLogger('sentry.webhooks')
+
+
+class GitHubAppsEndpoint(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    @method_decorator(csrf_exempt)
+    def dispatch(self, request, *args, **kwargs):
+        return super(GitHubAppsEndpoint, self).dispatch(request, *args, **kwargs)
+
+    def get_secret(self):
+        return options.get('github.webhook-secret')
+
+    def is_valid_signature(self, method, body, secret, signature):
+        if method == 'sha1':
+            mod = hashlib.sha1
+        else:
+            raise NotImplementedError('signature method %s is not supported' % (method, ))
```

Comment:
Style nit, this can just be written 

```py
if method != 'sha1':
    raise NotImplementedError('signature method %s is not supported' % (method, ))

mod = hashlib.sha1

```